exasol/python_extension_common/deployment/language_container_deployer.py

```diff
@@ -180,20 +193,18 @@ def run(self, container_file: Optional[Path] = None,
             """)
             print(message)
 
-    def upload_container(self, container_file: Path,
-                         bucket_file_path: Optional[str] = None) -> None:
+    def upload_container(self, container_file: Path, upload_path: bfs.path.PathLike) -> None:
```

Comment:
This change breaks the publicly accessible interface. It's not needed. Certainly not in the scope of this ticket. Please revert it.

Reply:
I was reluctant to this - thanks for confirmation!
I will change back.

Reply:
OK fixed in latest push.